### PR TITLE
Upgrade to rxjs to 6

### PR DESCRIPTION
**Pull request by AlexKhymenko**
This pull request will make library compatible with rxjs6 natively without need of rxjs compat

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,5 +9,4 @@ src/dropdown/*.js.map
 index.js
 index.js.map
 .vscode/*
-dist/*
```
Comment (martin-g) [inline .gitignore]: I think `dist/*` should be ignored.

File: package.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
     "rxjs": "^5.1.0",
```
Comment (martin-g) [inline package.json]: Why this is not updated to RxJS 6.1.0 ? I thought this is the main idea in this PR

**Comment by AlexKhymenko** [comment]
You are right changed it. Kind of in a rush.

File: package.json [hunk 3]
```diff
@@ -48,10 +48,10 @@
     "ng-packagr": "^2.2.0",
     "protractor": "^5.3.0",
     "rimraf": "^2.6.1",
-    "rxjs": "^5.1.0",
+    "rxjs": "6.0.0",
```
Comment (martin-g) [inline package.json]: Latest is `6.1.0`

**Pull request merged**